plugins/woocommerce/includes/wc-conditional-functions.php

```diff
@@ -281,6 +295,18 @@ function is_lost_password_page() {
 	}
 }
 
+if ( ! function_exists( 'is_admin_settings_page' ) ) {
+
+	/**
+	 * is_admin_settings_page - Returns true when viewing the admin settings page.
+	 *
+	 * @return bool
+	 */
+	function is_admin_settings_page() {
```

Comment:
@jorgeatorres @albarin : requesting your guidance on the new conditional functions naming :bow: